go/api/go_api/session.py

```diff
@@ -12,7 +13,8 @@ class SessionStore(SessionBase):
     """
     def __init__(self, session_key=None):
         super(SessionStore, self).__init__(session_key)
-        self.session_manager = vumi_api().session_manager
+        self.session_manager = SessionManager(
+            get_redis_manager().sub_manager('session_manager'))
```

Comment:
There's currently no way to create a `VumiApi` without also creating all the Riak-related things. That's fine for the common case when we actually want all those things, but not so much for this case where we don't.